docs/ServerSide.md

```diff
@@ -4,55 +4,63 @@ title: Server-side Editing
+You can configure the InterSystems ObjectScript extension to edit code directly on the server, using the [multi-root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) VS Code feature. This type of configuration is useful in cases where source code is stored in a Source Code Management (SCM) product interfaced to the server. For example you might already be using the Source Control menu in InterSystems Studio or Portal, implemented by a source control class that extends `%Studio.SourceControl.Base`.
 
 First configure the `intersystems.servers` entry for your server, as described in [Configuration](../configuration).
 
-Next create a workspace for editing code direct on the server:
+Next create a workspace for editing code directly on the server:
 
-1. Open VS Code. If a folder or workspace is already open, close it, for example by pressing <kbd>Ctrl/Cmd</kbd>+<kbd>K</kbd>, releasing that keypair, then pressing <kbd>F</kbd>.
+1. Open VS Code. You must perform the following steps starting with no folder or workspace open, so if a folder or workspace is already open, close it.
+1. Open the Explorer view if it is not already visible.
+1. Click the button labeled **Choose Server and Namespace** in the Explorer view, as shown in the following screen shot:
 
-2. Open the Explorer view (<kbd>Ctrl/Cmd</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd>) if it is not already visible.
+    ![Explorer view.](../assets/images/ss-explorer-view.png "explorer view")
+1. Pick a server from the list, or click the **+** sign to create a new server configuration:
 
-3. When the button labeled "Choose Server and Namespace" appears, click it.
+   ![Choose a server.](../assets/images/ss-choose-server.png "choose a server")
+1. Enter credentials if prompted.
+1. Pick a namespace from the list retrieved from the target server:
 
-4. Pick a server from your `intersystems.servers` settings object, or create a new server configuration.
+   ![Choose a namespace.](../assets/images/ss-choose-namespace.png "choose a namespace")
+1. Pick an access mode from the list:
 
-5. Enter credentials if prompted.
+   ![Choose an access type.](../assets/images/ss-access-type.png "choose an access type")
+1. If you want to reopen this workspace in the future, use the command **File > Save Workspace As...** to save it as a `.code-workspace` file.
 
-6. Pick a namespace from the list retrieved from the target server.
+Note that the ObjectScript button is not vivible in the Activity Bar, because the files listed in the Explorer view are all on the server, so it is not needed for this configuration.
```

Comment:
```suggestion
Note that the ObjectScript button is not visible in the Activity Bar, because the files listed in the Explorer view are all on the server, so it is not needed for this configuration.
```